dataclients/kubernetes/ingress.go

```diff
@@ -255,15 +255,11 @@ func annotationPredicate(m *definitions.Metadata) string {
 }
 
 // parse routes annotation
-func extraRoutes(m *definitions.Metadata, logger *logger) []*eskip.Route {
+func extraRoutes(m *definitions.Metadata) []*eskip.Route {
 	var extraRoutes []*eskip.Route
 	annotationRoutes := m.Annotations[definitions.IngressRoutesAnnotation]
 	if annotationRoutes != "" {
-		var err error
-		extraRoutes, err = eskip.Parse(annotationRoutes)
-		if err != nil {
-			logger.Errorf("Failed to parse routes from %s, skipping: %v", definitions.IngressRoutesAnnotation, err)
-		}
+		extraRoutes, _ = eskip.Parse(annotationRoutes) // We ignore the error here because it should be handled by the validator object
```

Comment:
Do we need to do the same change in `func annotationFilter`?

Reply:
Not yet, because in annotationFilter we also deal with `zalando.org/ratelimit` which we don't validate yet.

Reply:
1. deployments without validator webhook will still work because we depend on ingressValidator object which is already part of dataclient.
2. Currently it should migrate safely as we only remove logging from this function which is swapped by logging from Validate function.